packages/spec_cli/lib/src/io.dart

```diff
@@ -15,7 +15,8 @@ final $fileChange = StreamProvider<void>((ref) {
 
   final dir = ref.watch($workingDirectory);
   final libChange = Directory(join(dir.path, 'lib')).watch(recursive: true);
-  final testChange = Directory(join(dir.path, 'test')).watch(recursive: true);
+  final testChange =
```

Comment:
What's the reasoning for restarting tests when golden changes?

Reply:
It's opposite. So probably the name is wrong :D 
It removes golden files from observing so test won't restart on the golden file changes (unless you move or remove them). this protects against an endless loop.

Reply:
Oh my bad. Sounds good then